docs/Configuration.md

```diff
@@ -0,0 +1,18 @@
+# Configuration
+
+In order to set up your application you should create `config.js` file with all your configs and put it in the root:
```

Comment:
It might be a good idea to mention that this can also be put in `public/` folder (as `/public/config.js`).